linkerd/app/inbound/src/policy.rs

```diff
@@ -52,10 +56,15 @@ pub struct AllowPolicy {
 pub struct ServerPermit {
     pub dst: OrigDstAddr,
     pub protocol: Protocol,
-
     pub labels: ServerAuthzLabels,
 }
 
+#[derive(Clone, Debug, PartialEq, Eq, Hash)]
+pub struct RoutePermit {
+    pub dst: OrigDstAddr,
+    pub labels: RouteAuthzLabels,
+}
```

Comment:
might be nice to have comments on these explaining the relationship between `ServerPermit` and `RoutePermit`? nbd though.